--- apache_1.3.27/src/modules/standard/mod_auth_db.c.orig	Fri Feb 14 23:27:47 2003
+++ apache_1.3.27/src/modules/standard/mod_auth_db.c	Fri Feb 14 23:32:14 2003
@@ -107,6 +107,9 @@
 #endif
 #if (DB_VERSION_MAJOR == 4)
 #define DB4
+#if (DB_VERSION_MINOR >= 1)
+#define DB4_1
+#endif
 #endif
 #endif
 
@@ -170,7 +173,10 @@
     q.data = user;
     q.size = strlen(q.data);
 
-#if defined(DB3) || defined(DB4)
+#if defined(DB4_1)
+    if (   db_create(&f, NULL, 0) != 0
+	|| f->open(f, NULL, auth_dbpwfile, NULL, DB_HASH, DB_RDONLY, 0664) != 0) {
+#elif defined(DB3) || defined(DB4)
     if (   db_create(&f, NULL, 0) != 0 
         || f->open(f, auth_dbpwfile, NULL, DB_HASH, DB_RDONLY, 0664) != 0) {
 #elif defined(DB2)
