diff -Naurd build-tree.orig/apache_1.3.34/src/support/check_forensic build-tree/apache_1.3.34/src/support/check_forensic
--- build-tree.orig/apache_1.3.34/src/support/check_forensic	2004-01-15 21:46:20.000000000 +0100
+++ build-tree/apache_1.3.34/src/support/check_forensic	2004-05-18 19:30:32.000000000 +0200
@@ -11,5 +11,5 @@
 grep + < /tmp/fc-all.$$ | cut -c2- | sort > /tmp/fc-in.$$
 grep -- - < /tmp/fc-all.$$ | cut -c2- | sort > /tmp/fc-out.$$
 # use -i instead of -I for GNU xargs
-join -v 1 /tmp/fc-in.$$ /tmp/fc-out.$$ | xargs -I xx egrep "^\\+xx" $F
+join -v 1 /tmp/fc-in.$$ /tmp/fc-out.$$ | xargs -ixx egrep "^\\+xx" $F
 rm /tmp/fc-all.$$ /tmp/fc-in.$$ /tmp/fc-out.$$
