diff -urNp -x '*.orig' apache_1.3.42.org/src/support/apxs.pl apache_1.3.42/src/support/apxs.pl
--- apache_1.3.42.org/src/support/apxs.pl	2023-02-02 22:42:09.429834914 +0100
+++ apache_1.3.42/src/support/apxs.pl	2023-02-02 22:42:09.649834915 +0100
@@ -68,6 +68,7 @@ my $opt_i = 0;
 my $opt_a = 0;
 my $opt_A = 0;
 my $opt_q = 0;
+my $opt_v = 0;
 
 #   default for DSO file extension 
 my $dso_ext = "so";
@@ -153,7 +154,7 @@ sub usage {
 
 #   option handling
 my $rc;
-($rc, @ARGV) = &Getopts("qn:gco:I+D+L+l+W+S+eiaA", @ARGV);
+($rc, @ARGV) = &Getopts("qn:gco:I+D+L+l+W+S+eiaAv", @ARGV);
 &usage if ($rc == 0);
 &usage if ($#ARGV == -1 and not $opt_g);
 &usage if (not $opt_q and not ($opt_g and $opt_n) and not $opt_i and not $opt_c and not $opt_e);
@@ -210,7 +211,9 @@ sub execute_cmds {
     my ($cmd, $rc);
 
     foreach $cmd (@cmds) {
-        print STDERR "$cmd\n";
+	if ($opt_v or not $opt_e) {
+	    print STDERR "$cmd\n";
+	}
         $rc = system("$cmd");
         if ($rc != 0) {
             printf(STDERR "apxs:Break: Command failed with rc=%d\n", $rc >> 8);
