--- apache_1.3.11/src/support/apxs.pl.apxs	Thu Dec 16 17:10:30 1999
+++ apache_1.3.11/src/support/apxs.pl	Tue Jan 25 01:19:16 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # Copyright 1999-2004 The Apache Software Foundation
 #
@@ -220,15 +220,15 @@
 ##  Initial DSO support check
 ##
 if (not -x "$CFG_SBINDIR/$CFG_TARGET") {
-    print STDERR "apxs:Error: $CFG_SBINDIR/$CFG_TARGET not found or not executable\n";
-    exit(1);
+    print STDERR "apxs:Warning: $CFG_SBINDIR/$CFG_TARGET not found or not executable\n";
+    print STDERR "apxs:Warning: Continuing anyway...\n";
 }
 if (not grep(/mod_so/, `$CFG_SBINDIR/$CFG_TARGET -l`)) {
-    print STDERR "apxs:Error: Sorry, no DSO support for Apache available\n";
-    print STDERR "apxs:Error: under your platform. Make sure the Apache\n";
-    print STDERR "apxs:Error: module mod_so is compiled into your server\n";
-    print STDERR "apxs:Error: binary `$CFG_SBINDIR/$CFG_TARGET'.\n";
-    exit(1);
+    print STDERR "apxs:Warning: Sorry, no DSO support for Apache available\n";
+    print STDERR "apxs:Warning: under your platform. Make sure the Apache\n";
+    print STDERR "apxs:Warning: module mod_so is compiled into your server\n";
+    print STDERR "apxs:Warning: binary `$CFG_SBINDIR/$CFG_TARGET'.\n";
+    print STDERR "apxs:Warning: Continuing anyway...\n";
 }
 
 ##
