--- apache_1.3.22/src/support/apxs.pl.orig	Fri Dec  7 20:32:18 2001
+++ apache_1.3.22/src/support/apxs.pl	Fri Dec  7 21:47:25 2001
@@ -111,6 +111,7 @@
 my $opt_a = 0;
 my $opt_A = 0;
 my $opt_q = 0;
+my $opt_v = 0;
 
 #   this subroutine is derived from Perl's getopts.pl with the enhancement of
 #   the "+" metacharater at the format string to allow a list to be build by
@@ -190,7 +191,7 @@
 
 #   option handling
 my $rc;
-($rc, @ARGV) = &Getopts("qn:gco:I+D+L+l+W+S+eiaA", @ARGV);
+($rc, @ARGV) = &Getopts("qn:gco:I+D+L+l+W+S+eiaAv", @ARGV);
 &usage if ($rc == 0);
 &usage if ($#ARGV == -1 and not $opt_g);
 &usage if (not $opt_q and not ($opt_g and $opt_n) and not $opt_i and not $opt_c and not $opt_e);
@@ -247,7 +248,9 @@
     my ($cmd, $rc);
 
     foreach $cmd (@cmds) {
-        print STDERR "$cmd\n";
+	if ($opt_v or not $opt_e) {
+	    print STDERR "$cmd\n";
+	}
         $rc = system("$cmd");
         if ($rc != 0) {
             printf(STDERR "apxs:Break: Command failed with rc=%d\n", $rc >> 8);
