--- apache_1.3.2/Makefile.tmpl.orig	Fri Sep 25 23:56:02 1998
+++ apache_1.3.2/Makefile.tmpl	Fri Sep 25 23:57:37 1998
@@ -317,10 +317,6 @@
 	-@if [ ".$(suexec)" = .1 ]; then \
 	    echo "$(INSTALL_PROGRAM) $(TOP)/$(SRC)/support/suexec $(root)$(sbindir)/suexec"; \
 	    $(INSTALL_PROGRAM) $(TOP)/$(SRC)/support/suexec $(root)$(sbindir)/suexec; \
-	    echo "chown root $(root)$(sbindir)/suexec"; \
-	    chown root $(root)$(sbindir)/suexec; \
-	    echo "chmod 4711 $(root)$(sbindir)/suexec"; \
-	    chmod 4711 $(root)$(sbindir)/suexec; \
 	    echo "$(INSTALL_DATA) $(TOP)/$(SRC)/support/suexec.8 $(root)$(mandir)/man8/suexec.8"; \
 	    $(INSTALL_DATA) $(TOP)/$(SRC)/support/suexec.8 $(root)$(mandir)/man8/suexec.8; \
 	fi
