diff -Nru apache_1.3.17/src/include/httpd.h apache_1.3.17.new/src/include/httpd.h
--- apache_1.3.17/src/include/httpd.h	Wed Jan 31 20:49:26 2001
+++ apache_1.3.17.new/src/include/httpd.h	Wed Jan 31 20:51:53 2001
@@ -377,7 +377,7 @@
  */
 #ifdef EAPI
 #ifndef EAPI_MM_CORE_PATH
-#define EAPI_MM_CORE_PATH "logs/mm"
+#define EAPI_MM_CORE_PATH "/var/run/apache/mm"
 #endif
 #ifndef EAPI_MM_CORE_MAXSIZE
 #define EAPI_MM_CORE_MAXSIZE 1024*1024*1 /* max. 1MB */
