diff -urNp -x '*.orig' apache_1.3.42.org/src/support/apxs.pl apache_1.3.42/src/support/apxs.pl
--- apache_1.3.42.org/src/support/apxs.pl	2006-07-12 10:16:05.000000000 +0200
+++ apache_1.3.42/src/support/apxs.pl	2023-02-02 22:41:54.629834912 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # Licensed to the Apache Software Foundation (ASF) under one or more
 # contributor license agreements.  See the NOTICE file distributed with
@@ -188,15 +188,14 @@ if (@opt_S) {
 ##
 if ($^O ne "MSWin32") {
 if (not -x "$CFG_SBINDIR/$CFG_TARGET") {
-    print STDERR "apxs:Error: $CFG_SBINDIR/$CFG_TARGET not found or not executable\n";
-    exit(1);
+    print STDERR "apxs:Note: $CFG_SBINDIR/$CFG_TARGET not found or not executable\n";
 }
-if (not grep(/mod_so/, `$CFG_SBINDIR/$CFG_TARGET -l`)) {
-    print STDERR "apxs:Error: Sorry, no DSO support for Apache available\n";
-    print STDERR "apxs:Error: under your platform. Make sure the Apache\n";
-    print STDERR "apxs:Error: module mod_so is compiled into your server\n";
-    print STDERR "apxs:Error: binary `$CFG_SBINDIR/$CFG_TARGET'.\n";
-    exit(1);
+else { if (not grep(/mod_so/, `$CFG_SBINDIR/$CFG_TARGET -l`)) {
+    print STDERR "apxs:Warning: Sorry, no DSO support for Apache available\n";
+    print STDERR "apxs:Warning: under your platform. Make sure the Apache\n";
+    print STDERR "apxs:Warning: module mod_so is compiled into your server\n";
+    print STDERR "apxs:Warning: binary `$CFG_SBINDIR/$CFG_TARGET'.\n";
+    print STDERR "apxs:Warning: Continuing anyway...\n"; }
 }
 }
 
