diff -Nur apache_1.3.41/errordocs.orig/400.shtml.en apache_1.3.41/errordocs/400.shtml.en
--- apache_1.3.41/errordocs.orig/400.shtml.en	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/400.shtml.en	2008-02-09 20:08:41.633400775 +0100
@@ -0,0 +1,5 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-1"-->
+<!--#set var="TITLE" value="Bad Request"-->
+<!--#include virtual="HEADER.shtml"-->
+Your browser sent a request that this server could not understand.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/400.shtml.po apache_1.3.41/errordocs/400.shtml.po
--- apache_1.3.41/errordocs.orig/400.shtml.po	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/400.shtml.po	2008-02-09 20:09:07.014847182 +0100
@@ -0,0 +1,6 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-2"-->
+<!--#set var="TITLE" value="Niepoprawne zapytanie"-->
+<!--#include virtual="HEADER.shtml"-->
+Twoja przegl±darka przes³a³a zapytanie, którego nie mo¿e zinterpretowaæ serwer.
+B³±d w sk³adni zapytania.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/401.shtml.en apache_1.3.41/errordocs/401.shtml.en
--- apache_1.3.41/errordocs.orig/401.shtml.en	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/401.shtml.en	2008-02-09 20:09:20.167596714 +0100
@@ -0,0 +1,7 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-1"-->
+<!--#set var="TITLE" value="Authorization Required"-->
+<!--#include virtual="HEADER.shtml"-->
+This server could not verify that you are authorized to access the document
+you requested. Either you supplied the wrong credentials (e.g. bad password)
+or your browser doesn't understand how to supply the credentials required.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/401.shtml.po apache_1.3.41/errordocs/401.shtml.po
--- apache_1.3.41/errordocs.orig/401.shtml.po	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/401.shtml.po	2008-02-09 20:09:39.976725572 +0100
@@ -0,0 +1,8 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-2"-->
+<!--#set var="TITLE" value="Brak autoryzacji"-->
+<!--#include virtual="HEADER.shtml"-->
+Zapytanie nie przesz³o pomy¶lnie procesu uwierzytelnienia.
+Serwer nie mo¿e zweryfikowaæ czy posiadasz autoryzacjê dostêpu do dokumentu,
+który ¿±dasz. Prawdopodobnie wprowadzono nieprawid³owe dane jak np. has³o lub
+twoja przegl±darka nie potrafi dostarczyæ w³a¶ciwych danych do autoryzacji.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/403.shtml.en apache_1.3.41/errordocs/403.shtml.en
--- apache_1.3.41/errordocs.orig/403.shtml.en	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/403.shtml.en	2008-02-09 20:10:12.322568857 +0100
@@ -0,0 +1,9 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-1"-->
+<!--#set var="TITLE" value="Forbidden"-->
+<!--#include virtual="HEADER.shtml"-->
+You don't have permission to access
+<strong>
+<!--#echo var="REQUEST_URI"-->
+</strong>
+on this server.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/403.shtml.po apache_1.3.41/errordocs/403.shtml.po
--- apache_1.3.41/errordocs.orig/403.shtml.po	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/403.shtml.po	2008-02-09 20:10:21.451089061 +0100
@@ -0,0 +1,9 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-2"-->
+<!--#set var="TITLE" value="Dostêp zabroniony"-->
+<!--#include virtual="HEADER.shtml"-->
+Zapytanie odrzucone przez serwer. Nie masz dostêpu do
+<strong>
+<!--#echo var="REQUEST_URI"-->
+</strong>
+na tym serwerze.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/404.shtml.en apache_1.3.41/errordocs/404.shtml.en
--- apache_1.3.41/errordocs.orig/404.shtml.en	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/404.shtml.en	2008-02-09 20:10:33.927800068 +0100
@@ -0,0 +1,9 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-1"-->
+<!--#set var="TITLE" value="File Not Found"-->
+<!--#include virtual="HEADER.shtml"-->
+The requested URL
+<strong>
+<!--#echo var="REQUEST_URI"-->
+</strong>
+was not found on this server.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/404.shtml.po apache_1.3.41/errordocs/404.shtml.po
--- apache_1.3.41/errordocs.orig/404.shtml.po	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/404.shtml.po	2008-02-09 20:10:45.972486456 +0100
@@ -0,0 +1,9 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-2"-->
+<!--#set var="TITLE" value="Plik nie istnieje"-->
+<!--#include virtual="HEADER.shtml"-->
+Podany w zapytaniu URL
+<strong>
+<!--#echo var="REQUEST_URI"-->
+</strong>
+nie zosta³ odnaleziony na tym serwerze.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/405.shtml.en apache_1.3.41/errordocs/405.shtml.en
--- apache_1.3.41/errordocs.orig/405.shtml.en	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/405.shtml.en	2008-02-09 20:17:05.006086351 +0100
@@ -0,0 +1,12 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-1"-->
+<!--#set var="TITLE" value="Method Not Allowed"-->
+<!--#include virtual="HEADER.shtml"-->
+The requested method
+<strong>
+<!--#echo var="REQUEST_METHOD"-->
+</strong>
+is not allowed for the URL
+<strong>
+<!--#echo var="REQUEST_URI"-->
+</strong>.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/405.shtml.po apache_1.3.41/errordocs/405.shtml.po
--- apache_1.3.41/errordocs.orig/405.shtml.po	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/405.shtml.po	2008-02-09 20:17:17.506798725 +0100
@@ -0,0 +1,12 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-2"-->
+<!--#set var="TITLE" value="Niedozwolona metoda"-->
+<!--#include virtual="HEADER.shtml"-->
+Metoda
+<strong>
+<!--#echo var="REQUEST_METHOD"-->
+</strong>
+nie jest obs³ugiwana przez wybrany URL
+<strong>
+<!--#echo var="REQUEST_URI"-->
+</strong>.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/406.shtml.en apache_1.3.41/errordocs/406.shtml.en
--- apache_1.3.41/errordocs.orig/406.shtml.en	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/406.shtml.en	2008-02-09 20:17:31.643604335 +0100
@@ -0,0 +1,9 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-1"-->
+<!--#set var="TITLE" value="Not Acceptable"-->
+<!--#include virtual="HEADER.shtml"-->
+An appropriate representation of the requested resource
+<strong>
+<!--#echo var="REQUEST_URI"-->
+</strong>
+could not be found on this server.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/406.shtml.po apache_1.3.41/errordocs/406.shtml.po
--- apache_1.3.41/errordocs.orig/406.shtml.po	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/406.shtml.po	2008-02-09 20:17:44.224321269 +0100
@@ -0,0 +1,9 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-2"-->
+<!--#set var="TITLE" value="Brak akceptacji"-->
+<!--#include virtual="HEADER.shtml"-->
+Odpowiednia reprezentacja ¿±danego zasobu
+<strong>
+<!--#echo var="REQUEST_URI"-->
+</strong>
+nie zosta³a znaleziona przez serwer.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/408.shtml.en apache_1.3.41/errordocs/408.shtml.en
--- apache_1.3.41/errordocs.orig/408.shtml.en	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/408.shtml.en	2008-02-09 20:17:55.140943373 +0100
@@ -0,0 +1,5 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-1"-->
+<!--#set var="TITLE" value="Request Time-out"-->
+<!--#include virtual="HEADER.shtml"-->
+I'm tired of waiting for your request.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/408.shtml.po apache_1.3.41/errordocs/408.shtml.po
--- apache_1.3.41/errordocs.orig/408.shtml.po	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/408.shtml.po	2008-02-09 20:18:10.781834696 +0100
@@ -0,0 +1,5 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-2"-->
+<!--#set var="TITLE" value="Przekroczony czas oczekiwania"-->
+<!--#include virtual="HEADER.shtml"-->
+Przekroczony czas na wys³anie zapytania.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/410.shtml.en apache_1.3.41/errordocs/410.shtml.en
--- apache_1.3.41/errordocs.orig/410.shtml.en	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/410.shtml.en	2008-02-09 20:18:21.290433547 +0100
@@ -0,0 +1,10 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-1"-->
+<!--#set var="TITLE" value="Gone"-->
+<!--#include virtual="HEADER.shtml"-->
+The requested resource
+<strong>
+<!--#echo var="REQUEST_URI"-->
+</strong>
+is no longer available on this server and there is no forwarding address.
+Please remove all references to this resource.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/410.shtml.po apache_1.3.41/errordocs/410.shtml.po
--- apache_1.3.41/errordocs.orig/410.shtml.po	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/410.shtml.po	2008-02-09 20:18:32.895094859 +0100
@@ -0,0 +1,10 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-2"-->
+<!--#set var="TITLE" value="URI usuniête"-->
+<!--#include virtual="HEADER.shtml"-->
+Podany w zapytaniu URL
+<strong>
+<!--#echo var="REQUEST_URI"-->
+</strong>
+nie jest ju¿ dostêpny na tym serwerze. Proszê usun±æ wszystkie
+odwo³ania do tego adresu.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/411.shtml.en apache_1.3.41/errordocs/411.shtml.en
--- apache_1.3.41/errordocs.orig/411.shtml.en	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/411.shtml.en	2008-02-09 20:18:43.651707843 +0100
@@ -0,0 +1,9 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-1"-->
+<!--#set var="TITLE" value="Length Required"-->
+<!--#include virtual="HEADER.shtml"-->
+A request of the requested method
+<strong>
+<!--#echo var="REQUEST_METHOD"-->
+</strong>
+requires a valid Content-length.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/411.shtml.po apache_1.3.41/errordocs/411.shtml.po
--- apache_1.3.41/errordocs.orig/411.shtml.po	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/411.shtml.po	2008-02-09 20:18:52.816230100 +0100
@@ -0,0 +1,9 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-2"-->
+<!--#set var="TITLE" value="Wymagana d³ugo¶æ"-->
+<!--#include virtual="HEADER.shtml"-->
+Wybrana metoda
+<strong>
+<!--#echo var="REQUEST_METHOD"-->
+</strong>
+wymaga poprawnego nag³ówka Content-length.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/414.shtml.en apache_1.3.41/errordocs/414.shtml.en
--- apache_1.3.41/errordocs.orig/414.shtml.en	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/414.shtml.en	2008-02-09 20:19:02.252767858 +0100
@@ -0,0 +1,5 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-1"-->
+<!--#set var="TITLE" value="Request-URI Too Large"-->
+<!--#include virtual="HEADER.shtml"-->
+The requested URL's length exeeds the capacity limit for this server.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/414.shtml.po apache_1.3.41/errordocs/414.shtml.po
--- apache_1.3.41/errordocs.orig/414.shtml.po	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/414.shtml.po	2008-02-09 20:19:11.513295585 +0100
@@ -0,0 +1,5 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-2"-->
+<!--#set var="TITLE" value="Zbyt d³ugi URI"-->
+<!--#include virtual="HEADER.shtml"-->
+D³ugo¶æ wybranego URL-a przekracza zakres zdefiniowany na tym serwerze.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/500.shtml.en apache_1.3.41/errordocs/500.shtml.en
--- apache_1.3.41/errordocs.orig/500.shtml.en	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/500.shtml.en	2008-02-09 20:19:24.034009100 +0100
@@ -0,0 +1,14 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-1"-->
+<!--#set var="TITLE" value="Internal Server Error"-->
+<!--#include virtual="HEADER.shtml"-->
+The server encountered an internal error or misconfiguration and was unable
+to complete your request.
+</p>
+<p>
+<!--#if expr="$REDIRECT_ERROR_NOTES" -->
+Error message:
+<br /><!--#echo encoding="none" var="REDIRECT_ERROR_NOTES" -->
+<!--#else -->
+Either the server is overloaded or there was an error in a CGI script.
+<!--#endif -->
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/500.shtml.po apache_1.3.41/errordocs/500.shtml.po
--- apache_1.3.41/errordocs.orig/500.shtml.po	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/500.shtml.po	2008-02-09 20:19:37.602782340 +0100
@@ -0,0 +1,13 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-2"-->
+<!--#set var="TITLE" value="Wewnêtrzny b³±d serwera"-->
+<!--#include virtual="HEADER.shtml"-->
+Wewnêtrzny b³±d serwera lub b³±d konfiguracji.
+</p>
+<p>
+<!--#if expr="$REDIRECT_ERROR_NOTES" -->
+Komunikat b³êdu:
+<br /><!--#echo encoding="none" var="REDIRECT_ERROR_NOTES" -->
+<!--#else -->
+Serwer jest przeci±¿ony albo wyst±pi³ b³±d w skrypcie CGI.
+<!--#endif -->
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/503.shtml.en apache_1.3.41/errordocs/503.shtml.en
--- apache_1.3.41/errordocs.orig/503.shtml.en	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/503.shtml.en	2008-02-09 20:19:56.143838935 +0100
@@ -0,0 +1,6 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-1"-->
+<!--#set var="TITLE" value="Service Temporarily Unavailable"-->
+<!--#include virtual="HEADER.shtml"-->
+The server is temporarily unable to service your request due to
+maintenance downtime or capacity problems. Please try again later.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/503.shtml.po apache_1.3.41/errordocs/503.shtml.po
--- apache_1.3.41/errordocs.orig/503.shtml.po	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/503.shtml.po	2008-02-09 20:20:09.744613998 +0100
@@ -0,0 +1,6 @@
+<!--#set var="CHARACTER_ENCODING" value="iso-8859-2"-->
+<!--#set var="TITLE" value="Us³ugi tymczasowo niedostêpne"-->
+<!--#include virtual="HEADER.shtml"-->
+Us³ugi serwera s± czasowo niedostêpne co mo¿e byæ spowodowane jego zatrzymaniem
+lub prze³adowaniem. Proszê spróbowaæ po³±czyæ siê pó¼niej.
+<!--#include virtual="FOOT.shtml"-->
diff -Nur apache_1.3.41/errordocs.orig/FOOT.shtml apache_1.3.41/errordocs/FOOT.shtml
--- apache_1.3.41/errordocs.orig/FOOT.shtml	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/FOOT.shtml	2008-02-09 19:33:56.270562720 +0100
@@ -0,0 +1,9 @@
+</p>
+<p><a href="mailto:<!--#echo var="SERVER_ADMIN" -->">Administrator</a></p>
+<h2>Error <!--#echo encoding="none" var="REDIRECT_STATUS" --></h2>
+<address>
+<a href="/"><!--#echo encoding="entity" var="SERVER_NAME" --></a><br />
+<span><!--#echo var="SERVER_SOFTWARE" --></span>
+</address>
+</body>
+</html>
diff -Nur apache_1.3.41/errordocs.orig/HEADER.shtml apache_1.3.41/errordocs/HEADER.shtml
--- apache_1.3.41/errordocs.orig/HEADER.shtml	1970-01-01 01:00:00.000000000 +0100
+++ apache_1.3.41/errordocs/HEADER.shtml	2008-02-09 19:45:07.336804605 +0100
@@ -0,0 +1,8 @@
+<html>
+<head>
+<meta http-equiv="content-type" content="text/html; <!--#echo var="CHARACTER_ENCODING" -->">
+<title><!--#echo var="TITLE"--></title>
+</head>
+<body bgcolor="#ffffff">
+<h1 align="center"><!--#echo var="TITLE"--></h1>
+<p>
