diff -urN apache_1.3.17/src/main/http_main.c apache_1.3.17-new/src/main/http_main.c
--- apache_1.3.17/src/main/http_main.c	Sun Apr  1 00:56:33 2001
+++ apache_1.3.17-new/src/main/http_main.c	Sun Apr  1 00:57:12 2001
@@ -5004,7 +5004,7 @@
     ap_setup_prelinked_modules();
 
     while ((c = getopt(argc, argv,
-				    "D:C:c:xXd:Ff:vVlLR:StTh"
+				    "D:C:c:xXd:Ff:vVlLR:StTh4"
 #ifdef DEBUG_SIGSTOP
 				    "Z:"
 #endif
