diff -urNp -x '*.orig' apache_1.3.42.org/src/modules/proxy/proxy_http.c apache_1.3.42/src/modules/proxy/proxy_http.c
--- apache_1.3.42.org/src/modules/proxy/proxy_http.c	2023-02-02 22:43:29.949834915 +0100
+++ apache_1.3.42/src/modules/proxy/proxy_http.c	2023-02-02 22:43:30.219834915 +0100
@@ -604,6 +604,8 @@ int ap_proxy_http_handler(request_rec *r
         ap_table_set(resp_hdrs, "URI", proxy_location_reverse_map(r, urlstr));
     if ((urlstr = ap_table_get(resp_hdrs, "Content-Location")) != NULL)
         ap_table_set(resp_hdrs, "Content-Location", proxy_location_reverse_map(r, urlstr));
+    if ((urlstr = ap_table_get(resp_hdrs, "Content-Base")) != NULL)
+        ap_table_set(resp_hdrs, "Content-Base", proxy_location_reverse_map(r, urlstr));
 
 /* check if NoCache directive on this host */
   {
