--- apache_1.3.39/configure~	2007-12-26 21:20:04.000000000 +0200
+++ apache_1.3.39/configure	2007-12-26 21:22:23.219783498 +0200
@@ -458,10 +458,10 @@
             . $pldconf
             OOIFS="$IFS"  # most likely not needed: jmj
             IFS="$DIFS"   # ditto
-            for var in prefix exec_prefix bindir sbindir libexecdir mandir \
+            for var in prefix exec_prefix bindir sbindir libdir libexecdir mandir \
                        sysconfdir datadir iconsdir htdocsdir manualdir cgidir \
                        includedir localstatedir runtimedir logfiledir \
-                       proxycachedir; do
+                       proxycachedir infodir errordir installbuilddir; do
                 eval "val=\"\$$var\""
                 case $val in
                     *+ )
@@ -1077,10 +1077,10 @@
 ##
 OIFS="$IFS"
 IFS="$DIFS"
-for var in prefix exec_prefix bindir sbindir libexecdir mandir \
+for var in prefix exec_prefix bindir sbindir libdir libexecdir mandir \
            sysconfdir datadir iconsdir htdocsdir manualdir cgidir \
            includedir localstatedir runtimedir logfiledir \
-           proxycachedir suexec_docroot suexec_logexec ; do
+           proxycachedir infodir errordir installbuilddir suexec_docroot suexec_logexec ; do
     eval "val=\"\$$var\"";
     val=`echo $val | sed -e 's:\(.\)/*$:\1:'`
     eval "$var=\"$val\""
--- apache_1.3.39/config.layout~	2004-11-24 21:10:19.000000000 +0200
+++ apache_1.3.39/config.layout	2007-12-26 21:18:39.430942000 +0200
@@ -304,3 +304,27 @@
     proxycachedir: $localstatedir/cache/apache
 </Layout>
 
+# PLD layout...
+<Layout PLD>
+    prefix:        /usr
+    exec_prefix:   ${prefix}
+    bindir:        ${exec_prefix}/bin
+    sbindir:       ${exec_prefix}/sbin
+    libdir:        ${exec_prefix}/lib
+    libexecdir:    ${libdir}/apache1
+    mandir:        /usr/share/man
+    infodir:       /usr/share/info
+    sysconfdir:    /etc/apache
+    datadir:       ${prefix}/share/apache1
+    iconsdir:      ${prefix}/share/apache-icons
+    htdocsdir:     ${datadir}/html
+    manualdir:     ${datadir}/manual
+    errordir:      ${datadir}/errordocs
+    cgidir:        ${prefix}/lib/cgi-bin/apache1
+    includedir:    ${prefix}/include/apache1
+    localstatedir: /var/lib/apache
+    runtimedir:    /var/run
+    logfiledir:    /var/log/apache
+    proxycachedir: /var/cache/apache
+    installbuilddir: ${libexecdir}/build
+</Layout>
--- apache_1.3.14.orig/src/main/http_main.c	Thu Sep 28 15:32:36 2000
+++ apache_1.3.14/src/main/http_main.c	Sat Nov 11 20:39:48 2000
@@ -432,6 +432,9 @@
     else {
 	ap_add_version_component(SERVER_BASEVERSION " (" PLATFORM ")");
     }
+
+    ap_add_version_component(" (PLD/Linux)");
+    
     /*
      * Lock the server_version string if we're not displaying
      * the full set of tokens
