diff -urN apache_1.3.14.orig/config.layout apache_1.3.14/config.layout
--- apache_1.3.14.orig/config.layout	Tue Sep 26 01:44:55 2000
+++ apache_1.3.14/config.layout	Sat Nov 11 20:41:04 2000
@@ -249,3 +249,22 @@
     proxycachedir: $localstatedir/cache/apache
 </Layout>
 
+# PLD layout...
+<Layout PLD>
+    prefix:        /usr
+    exec_prefix:   $prefix
+    bindir:        $prefix/bin
+    sbindir:       $prefix/sbin
+    libexecdir:    $prefix/lib/apache
+    mandir:        $prefix/share/man
+    sysconfdir:    /etc/httpd
+    datadir:       /home/services/httpd
+    iconsdir:      $datadir/icons
+    htdocsdir:     $datadir/html
+    cgidir:        $datadir/cgi-bin
+    includedir:    $prefix/include/apache
+    localstatedir: /var
+    runtimedir:    $localstatedir/run
+    logfiledir:    $localstatedir/log/httpd
+    proxycachedir: $localstatedir/cache/www/apache
+</Layout>     
diff -urN apache_1.3.14.orig/src/main/http_main.c apache_1.3.14/src/main/http_main.c
--- apache_1.3.14.orig/src/main/http_main.c	Thu Sep 28 15:32:36 2000
+++ apache_1.3.14/src/main/http_main.c	Sat Nov 11 20:39:48 2000
@@ -432,6 +432,9 @@
     else {
 	ap_add_version_component(SERVER_BASEVERSION " (" PLATFORM ")");
     }
+
+    ap_add_version_component(" (PLD/Linux)");
+    
     /*
      * Lock the server_version string if we're not displaying
      * the full set of tokens
