diff -Nru apache_1.3.12/src/support/apxs.pl apache_1.3.12.new/src/support/apxs.pl
--- apache_1.3.12/src/support/apxs.pl	Mon Jun 12 18:37:55 2000
+++ apache_1.3.12.new/src/support/apxs.pl	Mon Jun 12 18:39:53 2000
@@ -424,7 +424,7 @@
         my $t = $f;
         $t =~ s|^.+/([^/]+)$|$1|;
         if ($opt_i) {
-            push(@cmds, "cp $f $CFG_LIBEXECDIR/$t");
+            push(@cmds, "cp -f $f $CFG_LIBEXECDIR/$t");
             push(@cmds, "chmod 755 $CFG_LIBEXECDIR/$t");
         }
 
@@ -512,9 +512,9 @@
             if (open(FP, ">$CFG_SYSCONFDIR/$CFG_TARGET.conf.new")) {
                 print FP $content;
                 close(FP);
-                system("cp $CFG_SYSCONFDIR/$CFG_TARGET.conf $CFG_SYSCONFDIR/$CFG_TARGET.conf.bak && " .
-                       "cp $CFG_SYSCONFDIR/$CFG_TARGET.conf.new $CFG_SYSCONFDIR/$CFG_TARGET.conf && " .
-                       "rm $CFG_SYSCONFDIR/$CFG_TARGET.conf.new");
+                system("cp -f $CFG_SYSCONFDIR/$CFG_TARGET.conf $CFG_SYSCONFDIR/$CFG_TARGET.conf.bak && " .
+                       "cp -f $CFG_SYSCONFDIR/$CFG_TARGET.conf.new $CFG_SYSCONFDIR/$CFG_TARGET.conf && " .
+                       "rm -f $CFG_SYSCONFDIR/$CFG_TARGET.conf.new");
             } else {
                 print STDERR "apxs:Error: unable to open configuration file\n";
             }
