diff -Nru apache_1.3.9/errordocs/400.shtml apache_1.3.9.new/errordocs/400.shtml
--- apache_1.3.9/errordocs/400.shtml	Thu Jan  1 01:00:00 1970
+++ apache_1.3.9.new/errordocs/400.shtml	Wed Jul 15 13:54:20 1998
@@ -0,0 +1,24 @@
+<!--#set var="title" value="Bad Request : Niepoprawne zapytanie"-->
+<!--#include virtual="HEADER.shtml"-->
+<H2 ALIGN="center">
+[<!--#echo var="REDIRECT_STATUS"-->] Bad Request : Niepoprawne zapytanie
+</H2>
+<HR>
+<CENTER>
+<TABLE>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="english"-->" WIDTH=26 HIGHT=19 ALT="English">
+</TD><TD>
+Your browser sent a request that this server could not understand.
+</TD></TR><TR><TD></TD><TD><HR></TD></TR>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="polish"-->" WIDTH=26 HIGHT=19 ALT="Polish">
+</TD><TD>
+Twoja przegl±darka przes³a³a zapytanie, które nie mo¿e zinterpretowaæ serwer.
+B³±d w sk³adni zapytania.
+</TD></TR>
+</TABLE>
+</CENTER>
+<P>
+<HR>
+<!--#include virtual="FOOT.shtml"--> 
diff -Nru apache_1.3.9/errordocs/401.shtml apache_1.3.9.new/errordocs/401.shtml
--- apache_1.3.9/errordocs/401.shtml	Thu Jan  1 01:00:00 1970
+++ apache_1.3.9.new/errordocs/401.shtml	Wed Jul 15 13:54:30 1998
@@ -0,0 +1,28 @@
+<!--#set var="title" value="Authorization Required : Brak autoryzacji"-->
+<!--#include virtual="HEADER.shtml"-->
+<H2 ALIGN="center">
+[<!--#echo var="REDIRECT_STATUS"-->] Authorization Required : Brak autoryzacji
+</H2>
+<HR>
+<CENTER>
+<TABLE>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="english"-->" WIDTH=26 HIGHT=19 ALT="English">
+</TD><TD>
+This server could not verify that you are authorized to access the document
+you requested. Either you supplied the wrong credentials (e.g. bad password)
+or your browser doesn't understand how to supply the credentials required.
+</TD></TR><TR><TD></TD><TD><HR></TD></TR>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="polish"-->" WIDTH=26 HIGHT=19 ALT="Polish">
+</TD><TD>
+Zapytanie nie przesz³o pomy¶lnie procesu uwierzytelnienia.
+Serwer nie mo¿e zweryfikowaæ czy posiadasz autoryzacjê dostêpu do dokumentu,
+który ¿±dasz. Prawdopodobnie wprowadzi³es nieprawid³owe dane jak np. has³o lub 
+twoja przegl±darka nie potrafi dostarczyæ w³a¶ciwych danych do autoryzacji.
+</TD></TR>
+</TABLE>
+</CENTER>
+<P>
+<HR>
+<!--#include virtual="FOOT.shtml"-->
diff -Nru apache_1.3.9/errordocs/403.shtml apache_1.3.9.new/errordocs/403.shtml
--- apache_1.3.9/errordocs/403.shtml	Thu Jan  1 01:00:00 1970
+++ apache_1.3.9.new/errordocs/403.shtml	Wed Jul 15 13:54:52 1998
@@ -0,0 +1,31 @@
+<!--#set var="title" value="Forbidden : Dostêp zabroniony"-->
+<!--#include virtual="HEADER.shtml"-->
+<H2 ALIGN="center">
+[<!--#echo var="REDIRECT_STATUS"-->] Forbidden : Dostêp zabroniony
+</H2>
+<HR>
+<CENTER>
+<TABLE>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="english"-->" WIDTH=26 HIGHT=19 ALT="English">
+</TD><TD>
+You don't have permission to access 
+<STRONG>
+<!--#echo var="REQUEST_URI"-->
+</STRONG>
+on this server.
+</TD></TR><TR><TD></TD><TD><HR></TD></TR>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="polish"-->" WIDTH=26 HIGHT=19 ALT="Polish">
+</TD><TD>
+Zapytanie odrzucone przez serwer. Nie masz dostêpu do 
+<STRONG>
+<!--#echo var="REQUEST_URI"-->
+</STRONG>
+na tym serwerze.
+</TD></TR>
+</TABLE>
+</CENTER>
+<P>
+<HR>
+<!--#include virtual="FOOT.shtml"-->
diff -Nru apache_1.3.9/errordocs/404.shtml apache_1.3.9.new/errordocs/404.shtml
--- apache_1.3.9/errordocs/404.shtml	Thu Jan  1 01:00:00 1970
+++ apache_1.3.9.new/errordocs/404.shtml	Wed Jul 15 13:54:58 1998
@@ -0,0 +1,31 @@
+<!--#set var="title" value="File Not Found : Plik nie istnieje"-->
+<!--#include virtual="HEADER.shtml"-->
+<H2 ALIGN="center">
+[<!--#echo var="REDIRECT_STATUS"-->] File Not Found : Plik nie istnieje
+</H2>
+<HR>
+<CENTER>
+<TABLE>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="english"-->" WIDTH=26 HIGHT=19 ALT="English">
+</TD><TD>
+The requested URL  
+<STRONG>
+<!--#echo var="REQUEST_URI"-->
+</STRONG>
+was not found on this server.
+</TD></TR><TR><TD></TD><TD><HR></TD></TR>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="polish"-->" WIDTH=26 HIGHT=19 ALT="Polish">
+</TD><TD>
+Podany w zapytaniu URL  
+<STRONG>
+<!--#echo var="REQUEST_URI"-->
+</STRONG>
+nie zosta³ odnaleziony na tym serwerze.
+</TD></TR>
+</TABLE>
+</CENTER>
+<P>
+<HR>
+<!--#include virtual="FOOT.shtml"-->
diff -Nru apache_1.3.9/errordocs/405.shtml apache_1.3.9.new/errordocs/405.shtml
--- apache_1.3.9/errordocs/405.shtml	Thu Jan  1 01:00:00 1970
+++ apache_1.3.9.new/errordocs/405.shtml	Wed Jul 15 13:55:04 1998
@@ -0,0 +1,37 @@
+<!--#set var="title" value="Method Not Allowed : Niedozwolona metoda"-->
+<!--#include virtual="HEADER.shtml"-->
+<H2 ALIGN="center">
+[<!--#echo var="REDIRECT_STATUS"-->] Method Not Allowed : Niedozwolona metoda
+</H2>
+<HR>
+<CENTER>
+<TABLE>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="english"-->" WIDTH=26 HIGHT=19 ALT="English">
+</TD><TD>
+The requested method
+<STRONG>  
+<!--#echo var="REQUEST_METHOD"--> 
+</STRONG>
+is not allowed for the URL  
+<STRONG>
+<!--#echo var="REQUEST_URI"-->.
+</STRONG>
+</TD></TR><TR><TD></TD><TD><HR></TD></TR>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="polish"-->" WIDTH=26 HIGHT=19 ALT="English">
+</TD><TD>
+Metoda 
+<STRONG>
+<!--#echo var="REQUEST_METHOD"--> 
+</STRONG>
+nie jest obs³ugiwana przez wybrany URL 
+<STRONG>
+<!--#echo var="REQUEST_URI"-->.
+</STRONG>
+</TD></TR>
+</TABLE>
+</CENTER>
+<P>
+<HR>
+<!--#include virtual="FOOT.shtml"--> 
diff -Nru apache_1.3.9/errordocs/406.shtml apache_1.3.9.new/errordocs/406.shtml
--- apache_1.3.9/errordocs/406.shtml	Thu Jan  1 01:00:00 1970
+++ apache_1.3.9.new/errordocs/406.shtml	Wed Jul 15 13:55:10 1998
@@ -0,0 +1,31 @@
+<!--#set var="title" value="Not Acceptable : Brak akceptacji"-->
+<!--#include virtual="HEADER.shtml"-->
+<H2 ALIGN="center">
+[<!--#echo var="REDIRECT_STATUS"-->] Not Acceptable : Brak akceptacji
+</H2>
+<HR>
+<CENTER>
+<TABLE>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="english"-->" WIDTH=26 HIGHT=19 ALT="English">
+</TD><TD>
+An appropriate representation of the requested resource 
+<STRONG> 
+<!--#echo var="REQUEST_URI"--> 
+</STRONG>
+could not be found on this server.
+</TD></TR><TR><TD></TD><TD><HR></TD></TR>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="polish"-->" WIDTH=26 HIGHT=19 ALT="Polish">
+</TD><TD>  
+Format podanego URL
+<STRONG>
+<!--#echo var="REQUEST_URI"--> 
+</STRONG>
+nie jest akceptowany przez serwer.
+</TD></TR>
+</TABLE>
+</CENTER>
+<P>
+<HR>
+<!--#include virtual="FOOT.shtml"--> 
diff -Nru apache_1.3.9/errordocs/408.shtml apache_1.3.9.new/errordocs/408.shtml
--- apache_1.3.9/errordocs/408.shtml	Thu Jan  1 01:00:00 1970
+++ apache_1.3.9.new/errordocs/408.shtml	Wed Jul 15 13:55:18 1998
@@ -0,0 +1,23 @@
+<!--#set var="title" value="Request Time-out : Przekroczony czas oczekiwania"-->
+<!--#include virtual="HEADER.shtml"-->
+<H2 ALIGN="center">
+[<!--#echo var="REDIRECT_STATUS"-->] Request Time-out : Przekroczony czas oczekiwania
+</H2>
+<HR>
+<CENTER>
+<TABLE>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="english"-->" WIDTH=26 HIGHT=19 ALT="English">
+</TD><TD>
+I'm tired of waiting for your request.
+</TD></TR><TR><TD></TD><TD><HR></TD></TR>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="polish"-->" WIDTH=26 HIGHT=19 ALT="Polish">
+</TD><TD> 
+Przekroczony czas na przygotowanie zapytania.
+</TD></TR>
+</TABLE>
+</CENTER> 
+<P>
+<HR>
+<!--#include virtual="FOOT.shtml"--> 
diff -Nru apache_1.3.9/errordocs/410.shtml apache_1.3.9.new/errordocs/410.shtml
--- apache_1.3.9/errordocs/410.shtml	Thu Jan  1 01:00:00 1970
+++ apache_1.3.9.new/errordocs/410.shtml	Wed Jul 15 13:55:22 1998
@@ -0,0 +1,33 @@
+<!--#set var="title" value="Gone : URI usuniête"-->
+<!--#include virtual="HEADER.shtml"-->
+<H2 ALIGN="center">
+[<!--#echo var="REDIRECT_STATUS"-->] Gone : URI usuniête
+</H2>
+<HR>
+<CENTER>
+<TABLE>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="english"-->" WIDTH=26 HIGHT=19 ALT="English">
+</TD><TD>
+The requested resource 
+<STRONG>
+<!--#echo var="REQUEST_URI"--> 
+</STRONG>
+is no longer available on this server and there is no forwarding address. 
+Please remove all references to this resource.
+</TD></TR><TR><TD></TD><TD><HR></TD></TR>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="polish"-->" WIDTH=26 HIGHT=19 ALT="Polish">
+</TD><TD>
+Podany w zapytaniu URL
+<STRONG>
+<!--#echo var="REQUEST_URI"--> 
+</STRONG>
+jest ju¿ nie dostêpny na tym serwerze. Proszê usun±æ wszystkie 
+odwo³ania do tego URL.
+</TD></TR>
+</TABLE>
+</CENTER>
+<P>
+<HR>
+<!--#include virtual="FOOT.shtml"--> 
diff -Nru apache_1.3.9/errordocs/411.shtml apache_1.3.9.new/errordocs/411.shtml
--- apache_1.3.9/errordocs/411.shtml	Thu Jan  1 01:00:00 1970
+++ apache_1.3.9.new/errordocs/411.shtml	Wed Jul 15 13:55:26 1998
@@ -0,0 +1,31 @@
+<!--#set var="title" value="Length Required : Wymagana d³ugo¶æ"-->
+<!--#include virtual="HEADER.shtml"-->
+<H2 ALIGN="center">
+[<!--#echo var="REDIRECT_STATUS"-->] Length Required : Wymagana d³ugo¶æ
+</H2>
+<HR>
+<CENTER>
+<TABLE>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="english"-->" WIDTH=26 HIGHT=19 ALT="English">
+</TD><TD>
+A request of the requested method  
+<STRONG> 
+<!--#echo var="REQUEST_METHOD"--> 
+</STRONG>
+requires a valid Content-length.
+</TD></TR><TR><TD></TD><TD><HR></TD></TR>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="polish"-->" WIDTH=26 HIGHT=19 ALT="Polish">
+</TD><TD>  
+Wybrana metoda 
+<STRONG>
+<!--#echo var="REQUEST_METHOD"--> 
+</STRONG>
+wymaga poprawnego nag³ówka Content-length.
+</TD></TR>
+</TABLE>
+</CENTER>
+<P>
+<HR>
+<!--#include virtual="FOOT.shtml"--> 
diff -Nru apache_1.3.9/errordocs/414.shtml apache_1.3.9.new/errordocs/414.shtml
--- apache_1.3.9/errordocs/414.shtml	Thu Jan  1 01:00:00 1970
+++ apache_1.3.9.new/errordocs/414.shtml	Wed Jul 15 13:55:34 1998
@@ -0,0 +1,23 @@
+<!--#set var="title" value="Request-URI Too Large : Zbyt d³ugi URI"-->
+<!--#include virtual="HEADER.shtml"-->
+<H2 ALIGN="center">
+[<!--#echo var="REDIRECT_STATUS"-->] Request-URI Too Large : Zbyt d³ugi URI
+</H2>
+<HR>
+<CENTER>
+<TABLE>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="english"-->" WIDTH=26 HIGHT=19 ALT="English">
+</TD><TD>
+The requested URL's length exeeds the capacity limit for this server.
+</TD></TR><TR><TD></TD><TD><HR></TD></TR>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="polish"-->" WIDTH=26 HIGHT=19 ALT="Polish">
+</TD><TD>
+D³ugo¶æ wybranego URL przekracza zekres zdefiniowany na tym serwerze.
+</TD></TR>
+</TABLE>
+</CENTER>
+<P>
+<HR>
+<!--#include virtual="FOOT.shtml"--> 
diff -Nru apache_1.3.9/errordocs/500.shtml apache_1.3.9.new/errordocs/500.shtml
--- apache_1.3.9/errordocs/500.shtml	Thu Jan  1 01:00:00 1970
+++ apache_1.3.9.new/errordocs/500.shtml	Wed Jul 15 13:55:38 1998
@@ -0,0 +1,37 @@
+<!--#set var="title" value="Internal Server Error : Wewnêtrzny b³±d serwera"-->
+<!--#include virtual="HEADER.shtml"-->
+<H2 ALIGN="center">
+[<!--#echo var="REDIRECT_STATUS"-->] Internal Server Error : Wewnêtrzny b³±d serwera
+</H2>
+<HR>
+<CENTER>
+<TABLE>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="english"-->" WIDTH=26 HIGHT=19 ALT="English">
+</TD><TD>
+The server encountered an internal error or misconfiguration and was unable 
+to complete your request. Please contact the server administrator 
+<A HREF="mailto:<!--#echo var="SERVER_ADMIN"-->?subject=B³±d: [<!--#echo var="REDIRECT_STATUS"-->] 
+		<!--#echo var="title"-->, URL=<!--#echo var="REQUEST_URI"-->">
+<!--#echo var="SERVER_ADMIN"-->
+</A> 
+and inform them of the time the error occured and anything you might have 
+done that may have caused the error.
+</TD></TR><TR><TD></TD><TD><HR></TD></TR>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="polish"-->" WIDTH=26 HIGHT=19 ALT="Polish">
+</TD><TD> 
+Wewnêtrzny b³±d serwera (np. zawieszenie programu CGI) lub b³±d konfiguracji.
+Proszê skontaktowaæ siê z administratorem 
+<A HREF="mailto:<!--#echo var="SERVER_ADMIN"-->?subject=B³±d: [<!--#echo var="REDIRECT_STATUS"-->] 
+		<!--#echo var="title"-->, URL=<!--#echo var="REQUEST_URI"-->">
+<!--#echo var="SERVER_ADMIN"-->
+</A>  
+i poinformowaæ go o czasie pojawienia siê b³êdu wraz z podaniem innych informacji, 
+które mog³y byæ przyczyn± b³êdu.
+</TD></TR>
+</TABLE>
+</CENTER>
+<P>
+<HR>
+<!--#include virtual="FOOT.shtml"--> 
diff -Nru apache_1.3.9/errordocs/503.shtml apache_1.3.9.new/errordocs/503.shtml
--- apache_1.3.9/errordocs/503.shtml	Thu Jan  1 01:00:00 1970
+++ apache_1.3.9.new/errordocs/503.shtml	Wed Jul 15 13:55:48 1998
@@ -0,0 +1,25 @@
+<!--#set var="title" value="Service Temporarily Unavailable : Us³ugi tymczasowo niedostêpne"-->
+<!--#include virtual="HEADER.shtml"-->
+<H2 ALIGN="center">
+[<!--#echo var="REDIRECT_STATUS"-->] Service Temporarily Unavailable : Us³ugi tymczasowo niedostêpne
+</H2>
+<HR>
+<CENTER>
+<TABLE>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="english"-->" WIDTH=26 HIGHT=19 ALT="English">
+</TD><TD>
+The server is temporarily unable to service your request due to 
+maintenance downtime or capacity problems. Please try again later.
+</TD></TR><TR><TD></TD><TD><HR></TD></TR>
+<TR><TD VALIGN="top" ALIGN="left" WIDTH=50>
+<IMG SRC="<!--#echo var="polish"-->" WIDTH=26 HIGHT=19 ALT="Polish">
+</TD><TD>
+Us³ugi serwera s± czasowo niedostêpne co mo¿e byæ spowodowane jego zatrzymaniem 
+lub prze³adowaniem. Proszê spróbowaæ po³±czyæ siê pó¼niej.
+</TD></TR>
+</TABLE>
+</CENTER>
+<P>
+<HR>
+<!--#include virtual="FOOT.shtml"--> 
diff -Nru apache_1.3.9/errordocs/FOOT.shtml apache_1.3.9.new/errordocs/FOOT.shtml
--- apache_1.3.9/errordocs/FOOT.shtml	Thu Jan  1 01:00:00 1970
+++ apache_1.3.9.new/errordocs/FOOT.shtml	Sun Jan  3 17:11:22 1999
@@ -0,0 +1,30 @@
+<P>
+<!--#set var="Mail" value="/icons/email.gif"-->
+<!--#set var="Powered_by_Apache" value="/icons/apache_pb.gif"-->
+<!--#set var="Powered_by_Linux" value="/icons/linux_pwd.gif"-->
+<CENTER>
+<IMG SRC="<!--#echo var="Mail"-->" WIDTH=50 HIGHT=66 ALT="Mail to:">
+<BR>
+<A HREF="mailto:<!--#echo var="SERVER_ADMIN"-->?subject=Blad: [<!--#echo var="REDIRECT_STATUS"-->] 
+		<!--#echo var="title"-->, URL=<!--#echo var="REQUEST_URI"-->">
+<!--#echo var="SERVER_ADMIN"-->
+</A>
+<P> 
+<TABLE WIDTH=50%>
+<TR>
+<TD ALIGN="left">
+<IMG SRC="<!--#echo var="Powered_by_Apache"-->" WIDTH=259 HIGHT=32 ALT="Powered by Apache">
+</TD>
+<TD ALIGN="right">
+<IMG SRC="<!--#echo var="Powered_by_Linux"-->" WIDTH=196 HIGHT=49 ALT="Powered by Linux">
+</TD>
+</TR>
+</TABLE>
+<BR>
+<FONT SIZE=-3>
+data:<!--#echo var="DATE_LOCAL"-->,
+ostatnia modyfikacja: <!--#echo var="LAST_MODIFIED"-->
+</FONT>
+</CENTER>
+</BODY>
+</HTML>
diff -Nru apache_1.3.9/errordocs/HEADER.shtml apache_1.3.9.new/errordocs/HEADER.shtml
--- apache_1.3.9/errordocs/HEADER.shtml	Thu Jan  1 01:00:00 1970
+++ apache_1.3.9.new/errordocs/HEADER.shtml	Wed Jul 15 13:56:44 1998
@@ -0,0 +1,14 @@
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=ISO-8859-2">
+<TITLE>
+<!--#echo var="title"-->
+</TITLE>
+</HEAD>
+<BODY BGCOLOR="#FFFFFF">
+<!--#set var="english" value="/icons/gb.gif"-->
+<!--#set var="polish" value="/icons/poland.gif"-->
+<H2 ALIGN="center">
+[Error : B³±d]
+</H2>
+<P>
