diff -Nur apache_1.3.17.orig/src/modules/standard/mod_auth_dbm.c apache_1.3.17/src/modules/standard/mod_auth_dbm.c
--- apache_1.3.17.orig/src/modules/standard/mod_auth_dbm.c	Wed Feb  7 23:10:02 2001
+++ apache_1.3.17/src/modules/standard/mod_auth_dbm.c	Wed Feb  7 23:14:06 2001
@@ -81,7 +81,8 @@
 #define dbm_fetch sdbm_fetch
 #define dbm_close sdbm_close
 #else
-#include <ndbm.h>
+#define DB_DBM_HSEARCH 1
+#include <db.h>
 #endif
 
 /*
diff -Nur apache_1.3.17.orig/src/modules/standard/mod_rewrite.h apache_1.3.17/src/modules/standard/mod_rewrite.h
--- apache_1.3.17.orig/src/modules/standard/mod_rewrite.h	Thu Jan 18 23:31:44 2001
+++ apache_1.3.17/src/modules/standard/mod_rewrite.h	Wed Feb  7 23:15:22 2001
@@ -130,7 +130,8 @@
      * so we also need to know the file extension
      */
 #ifndef NO_DBM_REWRITEMAP
-#include <ndbm.h>
+#define DB_DBM_HSEARCH 1
+#include <db.h>
 #if defined(DBM_SUFFIX)
 #define NDBM_FILE_SUFFIX DBM_SUFFIX
 #elif defined(__FreeBSD__) || (defined(DB_LOCK) && defined(DB_SHMEM))
--- apache_1.3.23/src/helpers/find-dbm-lib.orig	Tue Jan 22 08:39:15 2002
+++ apache_1.3.23/src/helpers/find-dbm-lib	Wed Feb  6 21:34:50 2002
@@ -13,7 +13,9 @@
 	    *-linux*)
 		# many systems don't have -ldbm
 		DBM_LIB=""
-		if ./helpers/TestCompile lib ndbm dbm_open; then
+		if ./helpers/TestCompile lib db __db_ndbm_open; then
+		    DBM_LIB="-ldb"
+		elif ./helpers/TestCompile lib ndbm dbm_open; then
 		    DBM_LIB="-lndbm"
 		    if ./helpers/TestCompile lib db1 dbm_open; then
 			# Red Hat needs this; ndbm.h lives in db1
