--- apache_1.3.42/src/main/http_log.c~	2006-07-12 10:16:05.000000000 +0200
+++ apache_1.3.42/src/main/http_log.c	2012-11-12 22:36:48.615990057 +0100
@@ -487,7 +487,7 @@
        *      that may screw up scripts written to do something
        *      based on the last modification time of the pid file.
        */
-      ap_log_error(APLOG_MARK, APLOG_NOERRNO|APLOG_WARNING, NULL,
+      ap_log_error(APLOG_MARK, APLOG_NOERRNO|APLOG_WARNING, NULL, "%s",
 		   ap_psprintf(p,
 			       "pid file %s overwritten -- Unclean shutdown of previous Apache run?",
 			       fname)
