diff -urNp -x '*.orig' apache_1.3.42.org/src/include/hsregex.h apache_1.3.42/src/include/hsregex.h
--- apache_1.3.42.org/src/include/hsregex.h	2005-10-10 19:45:29.000000000 +0200
+++ apache_1.3.42/src/include/hsregex.h	2023-02-02 22:42:24.093168248 +0100
@@ -22,7 +22,7 @@ extern "C" {
 #define ap_private_extern
 #endif
 
-typedef off_t regoff_t;
+typedef int regoff_t;
 typedef struct {
 	int re_magic;
 	size_t re_nsub;		/* number of parenthesized subexpressions */
