diff -urN apache_1.3.14.orig/src/modules/standard/mod_auth_dbm.c apache_1.3.14/src/modules/standard/mod_auth_dbm.c
--- apache_1.3.14.orig/src/modules/standard/mod_auth_dbm.c	Sat Nov 11 22:58:04 2000
+++ apache_1.3.14/src/modules/standard/mod_auth_dbm.c	Sat Nov 11 22:59:52 2000
@@ -80,8 +80,12 @@
 #define dbm_fetch sdbm_fetch
 #define dbm_close sdbm_close
 #elif defined(__GLIBC__) && defined(__GLIBC_MINOR__) \
-    && __GLIBC__ >= 2 && __GLIBC_MINOR__ >= 1
+    && __GLIBC__ >= 2 && __GLIBC_MINOR__ == 1
 #include <db1/ndbm.h>
+#elif defined(__GLIBC__) && defined(__GLIBC_MINOR__) \
+    && __GLIBC__ >= 2 && __GLIBC_MINOR__ >= 2
+#define DB_DBM_HSEARCH 1
+#include <db.h>
 #else
 #include <ndbm.h>
 #endif
