diff -urNp -x '*.orig' apache_1.3.42.org/configure apache_1.3.42/configure
--- apache_1.3.42.org/configure	2023-02-02 22:43:53.016501583 +0100
+++ apache_1.3.42/configure	2023-02-02 22:43:53.329834916 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # Licensed to the Apache Software Foundation (ASF) under one or more
 # contributor license agreements.  See the NOTICE file distributed with
diff -urNp -x '*.orig' apache_1.3.42.org/src/Configure apache_1.3.42/src/Configure
--- apache_1.3.42.org/src/Configure	2023-02-02 22:43:53.066501583 +0100
+++ apache_1.3.42/src/Configure	2023-02-02 22:43:53.329834916 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # Licensed to the Apache Software Foundation (ASF) under one or more
 # contributor license agreements.  See the NOTICE file distributed with
@@ -210,7 +210,7 @@ fi
 ##
 DBM_LIB="-ldbm"
 DB_LIB="-ldb"
-SHELL="/bin/sh"
+SHELL="/bin/bash"
 SUBTARGET="target_static"
 SHLIB_SUFFIX_NAME=""
 SHLIB_SUFFIX_LIST=""
diff -urNp -x '*.orig' apache_1.3.42.org/src/helpers/TestCompile apache_1.3.42/src/helpers/TestCompile
--- apache_1.3.42.org/src/helpers/TestCompile	2006-07-12 10:16:05.000000000 +0200
+++ apache_1.3.42/src/helpers/TestCompile	2023-02-02 22:43:53.333168250 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # Licensed to the Apache Software Foundation (ASF) under one or more
 # contributor license agreements.  See the NOTICE file distributed with
